static/js/containers/CollectionDetailPage.js

```diff
@@ -112,10 +123,22 @@ class CollectionDetailPage extends React.Component {
         </div>
         <div className="tools">
           {
-            collection.is_admin &&
-            <a onClick={ showDialog.bind(this, DIALOGS.NEW_COLLECTION) }>
-              <i className="material-icons">settings</i>
-            </a>
+            collection.is_admin && [
+              <DropboxChooser
+                key="upload"
+                appKey={SETTINGS.dropbox_key}
+                success={this.handleUpload}
+                multiselect={true}
```

Comment:
Try adding this option here:  `extensions: ['video']`

This was on the old HTML template form and only allowed video files to be chosen.  With the new chooser I'm allowed to upload any file.